api/src/db/db.service.spec.ts

```diff
@@ -71,14 +76,97 @@ describe("DbService", () => {
     });
 
     it("can update get the latest document updated time", async () => {
-        const res: number = await service.getLatestUpdatedTime();
+        const res: number = await service.getLatestDocUpdatedTime();
 
-        expect(res).toBe(2);
+        expect(res).toBe(3);
     });
 
     it("can update get the oldest changelogEntry document updated time", async () => {
-        const res: number = await service.getOldestChangelogEntryUpdatedTime();
+        const res: number = await service.getOldestChangeTime();
 
         expect(res).toBe(1);
     });
+
+    it("can retrieve user's own document", async () => {
+        const res: any = await service.getDocs("user-public", {
+            groups: [],
+            types: [],
+            from: 0,
+        });
+
+        expect(res.docs[0]._id).toBe("user-public");
+        expect(res.docs.length).toBe(1);
+    });
+
+    it("can retrieve documents using one group selector", async () => {
+        const res: any = await service.getDocs("user-public", {
+            groups: ["group-public-content"],
+            types: ["post", "tag"],
+            from: 0,
+        });
+
+        expect(res.docs.length).toBe(4);
```

Comment:
I suggest adding assertions for the types of document so we know what's returned.